trunner/harness/psh.py

```diff
@@ -72,7 +72,15 @@ def __call__(self, result: TestResult) -> TestResult:
         if self.cmd is not None:
             self.dut.send(self.cmd + "\n")
             try:
-                self.dut.expect(f"{self.cmd}(\r+)\n")
+                # ash wraps up lines longer than 80 characters
```

Comment:
Please make the commit message shorter, `trunner: add capability to assert sending long commands due to ash line wrap up` -> `trunner: add capability to assert sending long commands in ash`